src/WidgetCustomization.tsx

```diff
@@ -32,6 +32,8 @@ type Props = {
   pubSub?: PubSub;
   /** Callback which will be called on menu section change */
   onMenuItemChange?: (menuItemId: string) => void;
+  /** Result of query */
+  result?: unknown;
```

Comment:
Just for better readability I suggest to rename it to `analysisResult`. What do You think ?